apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -524,6 +524,72 @@ describe('Trigger event - /v1/events/trigger (POST)', function () {
     const subscriberIds = messages.map((message) => message._subscriberId).filter(isUnique);
     expect(subscriberIds.length).to.equal(4);
   });
+
+  it('should not filter a message with correct payload', async function () {
```

Comment:
Maybe worth adding a couple of other steps here to make sure only the intended runs? Or to do a negated value to make sure it's not running? This test might pass even if the filter is not working or implemented. wdyt?

Reply:
Yeah this was what I thought but I think we had some communication problem in Discord, will fix this 😄